lib/msf/core/db_manager/async_callback.rb

```diff
@@ -0,0 +1,39 @@
+module Msf::DBManager::AsyncCallback
+
+  def create_async_callback(opts)
+    ::ActiveRecord::Base.connection_pool.with_connection do
+      # Disabled UUID checking, since we anticipate multiple callbacks from the same UUID
+      #if opts[:uuid] && !opts[:uuid].to_s.empty?
+      #  if Mdm::AsyncCallback.find_by(uuid: opts[:uuid])
+      #    raise ArgumentError.new("An async callback with this uuid already exists.")
+      #  end
+      #end
+
+      Mdm::AsyncCallback.create!(opts)
+    end
+  end
+
+  def async_callbacks(opts)
+    ::ActiveRecord::Base.connection_pool.with_connection do
+      if opts[:uuid] && !opts[:uuid].to_s.empty?
+        #return Array.wrap(Mdm::AsyncCallback.find_by :uuid => opts[:uuid] )
+        #return Array.wrap( Mdm::AsyncCallback.where ( uuid: opts[:uuid] ))
+        return Mdm::AsyncCallback.where( 'uuid' => opts[:uuid] )
```

Comment:
I see commented out lines above that use `Array.wrap` around the `Mdm::AsyncCallback` query. What happens when this is called with no results or a single result when connected to a remote data service?

Reply:
If the PR is going to be landed with AsyncCallback code present, but unused, would you add a TODO to the code to address the above concern?